apps/perf-test/src/scenarios/BaseButtonNew.tsx

```diff
@@ -1,6 +1,6 @@
 import * as React from 'react';
 import { Actionable } from '@uifabric/experiments';
 
-const scenario = <Actionable content="I am a button" />;
+const scenario = <Actionable>I am a button</Actionable>;
```

Comment:
Because we created the new `ActionableView` we can no longer use the `content` prop here.